worlds/sc2/ItemGroups.py

```diff
@@ -484,8 +486,8 @@ class ItemGroupNames:
 item_name_groups[ItemGroupNames.STARGATE_UNITS] = stargate_units = [
     ItemNames.PHOENIX, ItemNames.MIRAGE, ItemNames.CORSAIR,
     ItemNames.VOID_RAY, ItemNames.DESTROYER,
-    ItemNames.SCOUT, ItemNames.TEMPEST,
-    ItemNames.CARRIER, ItemNames.MOTHERSHIP,
+    ItemNames.SCOUT, ItemNames.MOTHERSHIP,
+    ItemNames.CARRIER, ItemNames.TEMPEST,
```

Comment:
(A Purifier variant of Carrier is planned for the future)

Reply:
I'm just going to separate them out on their own lines as the relationship is non-obvious to me. Whoever adds the new carrier variant item can reformat it as they wish.

Reply:
The other units there are put in lines as they're built in their buildings. Therefore it makes sense to follow that pattern for cosistency

Reply:
I do not see a logic connection between scout, tempest, and mothership, that excludes carriers, and don't see the need to copy the command-card layout exactly seeing as it's subject to change. Personally, the grouping I would consider intuitive is (carrier, tempest) as capital ships and scout and mothership separated / as miscellaneous.

Remember this currently isn't user-facing, and thus the styling isn't immediately important, but in future we need to provide some way for yaml writers to look these groups up. That will involve either pointing them straight at this file, in which case the grouping doesn't really make much sense to me; or it will involve some automated script to collect the groups into a list we can post somewhere, in which case order matters but indentation doesn't. In the latter case, the order I'd expect for these "ungrouped" starships is scout -> carrier -> tempest -> mothership or scout -> tempest -> carrier -> mothership, as that's roughly the "other" category organized by size. I could maybe see it worthwhile to put arbiter and oracle before this segment.

Also, are we going to have the mothership on the stargate indefinitely, or are there plans to put it on the nexus like in melee?

Reply:
Stared at it very hard. If tempest goes first in that line then I guess the exported version will still put carrier and tempest next to each other, which is good enough I guess. Updated.